.github/CODEOWNERS

```diff
@@ -0,0 +1,24 @@
+# Global configs
+.github/ @abhinavrastogi-harness @knagurski
+.eslintrc.js @abhinavrastogi-harness @knagurski
+.prettierrc.yml @abhinavrastogi-harness @knagurski
+
+# Common ownership for shared config files
+/packages/*/package.json @abhinavrastogi-harness @knagurski
+/apps/*/package.json @abhinavrastogi-harness @knagurski
+/packages/*/vite.config.ts @abhinavrastogi-harness @knagurski
+/apps/*/vite.config.ts @abhinavrastogi-harness @knagurski
+
+# Apps
+/apps/gitness/* @abhinavrastogi-harness @vardanbansal-harness
+/apps/design-system/* @knagurski
```

Comment:
maybe we don't need codeowners for `design-system` app as it's just the view previews. we're not shipping this anywhere. let's avoid unnecessary friction where possible

Reply:
Removed it.